portal/flatpak-portal.c

```diff
@@ -2969,7 +3002,7 @@ on_bus_acquired (GDBusConnection *connection,
 
   g_object_set_data_full (G_OBJECT (portal), "track-alive", GINT_TO_POINTER (42), skeleton_died_cb);
 
-  portal_flatpak_set_version (PORTAL_FLATPAK (portal), 7);
+  portal_flatpak_set_version (PORTAL_FLATPAK (portal), 8);
```

Comment:
I think you have update this here too https://github.com/flatpak/flatpak/blob/947d4a6ce445ce71a854e6747d88dd5016afe65a/tests/test-portal.c#L252